.travis.yml

```
@@ -3,10 +3,9 @@ os: osx
 osx_image: xcode10
 env:
   global:
-    # Code coverage key
-    - CC_TEST_REPORTER_ID=62a314af9e8f1580690d6032a2998b5d9756097bcb662a029aa2e97935fa7d45
     # Ruby setup
     - BUNDLE_GEMFILE=.ruby-dependencies/Gemfile
+    - secure: QiKA+JoGk6C7PFdphqS6k9hC6p1TO7cBBhoF3NMlGo2mlwxoVnagtUpoM5kmUW2PoG49LMGxQI/oerQMSqY/dJa0df9UlffMZTs6UIBdxbbeg7Rp/gh0DMMQQGG2mokBLM8Ivm/tFxlf8Y8CHHSWgBWjKTvnlX21nTvYgqqiCfcEmae/kJgBq9M/w9cQNXFnQJm6FsTrekESBlX4CzmyWxeLWVGEvgsXTwg6FsNVIZFuiSJPflLja5X7MzizZX73jvKRHCts49/ftO1TL6ii5DjuYBOzILg9Zlo01Y+2AgTYAX8dou+wUDbHfPsK66trB4wblDmqyB9dcWG+2oc8cMEGqyrIAFP5wEvuVqkujPHBBc7gYJNXBCFDS/Qy/Xmobj4YbSPnw9gnYK+Ba5nM89/zJOQUnozWcitlfkFnho+PMEYrciYnTOJsnT2MV/GGc1B7ccT8HrAkU8jQQgwh+i56TgzXR7KJ0g40eEKK98RLcSjrVOy35NInrnSy7WawpydseLOxRtjLZPnKLrAOtMJ2zbjx8xnAldzLxK9lwsEQGBBAcieDoJheEwUVIOEHW7gyIGOMBoqY8zLWdcpKgy/luOxLdXtI4IxvqMtHOtvkC1BbH6mAbfeBmmCAcHfDlZ9ldnR647kKr+TPvyKMHqvZX16E+UAOVEnJ9QNEiRc=
```

I would keep the comment about code coverage, otherwise it seems as though the secure variable is part of the Ruby setup.

```suggestion
    # Code coverage key
    - secure: QiKA+JoGk6C7PFdphqS6k9hC6p1TO7cBBhoF3NMlGo2mlwxoVnagtUpoM5kmUW2PoG49LMGxQI/oerQMSqY/dJa0df9UlffMZTs6UIBdxbbeg7Rp/gh0DMMQQGG2mokBLM8Ivm/tFxlf8Y8CHHSWgBWjKTvnlX21nTvYgqqiCfcEmae/kJgBq9M/w9cQNXFnQJm6FsTrekESBlX4CzmyWxeLWVGEvgsXTwg6FsNVIZFuiSJPflLja5X7MzizZX73jvKRHCts49/ftO1TL6ii5DjuYBOzILg9Zlo01Y+2AgTYAX8dou+wUDbHfPsK66trB4wblDmqyB9dcWG+2oc8cMEGqyrIAFP5wEvuVqkujPHBBc7gYJNXBCFDS/Qy/Xmobj4YbSPnw9gnYK+Ba5nM89/zJOQUnozWcitlfkFnho+PMEYrciYnTOJsnT2MV/GGc1B7ccT8HrAkU8jQQgwh+i56TgzXR7KJ0g40eEKK98RLcSjrVOy35NInrnSy7WawpydseLOxRtjLZPnKLrAOtMJ2zbjx8xnAldzLxK9lwsEQGBBAcieDoJheEwUVIOEHW7gyIGOMBoqY8zLWdcpKgy/luOxLdXtI4IxvqMtHOtvkC1BbH6mAbfeBmmCAcHfDlZ9ldnR647kKr+TPvyKMHqvZX16E+UAOVEnJ9QNEiRc=
```